02_Dataflow/Schemas/iotToBigQuery.json

```
@@ -7,18 +7,23 @@
     },
     {
       "mode": "NULLABLE",
-      "name": "timeStamp",
+      "name": "client_id",
       "type": "STRING"
     },
     {
       "mode": "NULLABLE",
-      "name": "client_id",
+      "name": "timeStamp",
```

La s tiene que ser minùscula 

```suggestion
      "name": "timestamp",
```